packages/opc-back-to-top/README.md

```diff
@@ -0,0 +1,127 @@
+# This is opc-back-to-top 👋
```

Comment:
rename to `This is opc-back-top-top component`

Reply:
```suggestion
# This is opc-back-to-top  component 👋
```
@anjneeksharma  What @diwanshi mentioned (-top-top) must be a typo. I suppose she meant what I have suggested

Reply:
yes that was a typo on my end